drivers/gpu/drm/msm/registers/gen_header.py

```diff
@@ -11,6 +11,7 @@
 import argparse
 import time
 import datetime
+import re
```

Comment:
In this patch the trailing tags list is broken with an empty line inbetween. Why?